File: C_Programing/Competitive/Problems_Solving/ASSIGNMENT_12/Q3.c
//WAP which accept  from user and count frequency  2 in it 
#include<stdio.h>

  
int CountTwo(int iNo)
{

  if(iNo<0)
  {
    iNo=-iNo;
  }

int iCount=0;
  int iDigit=0 ;
 while (iNo!=0)
 {
   iDigit=iNo%10;//Digit remainder check and forward in printf 
  
    if(iDigit==2)
   {
    iCount++;
   }    
   iNo=iNo/10;//540/54/4/5//Digit Seperate logic 
 }
 return  iCount;
   
}


int main()
{
   
      int iValue=0;
      int iRet=0;
    printf("Enter number  : ");
    scanf("%d",&iValue);


 iRet =CountTwo(iValue);
  
 printf("%d",iRet);

    return 0;
}

File: java_Programing/Competitive/Problems_Solving/ASSIGNMENT_19/Q3.java
//WAP to check whether  a number is divisible by 5 and 11 or not  ...

import java.util.Scanner;




class Logic
{
   
   void CheckDivisible(int iNo)
    {
      if(iNo % 5==0 && iNo % 11 ==0)  
      {
        System.out.println( iNo + " - is Divisible By 5 or 11 ");
      }
      else
      {
        System.out.println( iNo + " - is not  Divisible By 5 or 11 ");
      }
    }

}
class Q3
{
    public static void main(String A[] )
     {
        int iValue=0;
      
       
      Scanner sobj=new Scanner(System.in);

       System.out.println("Enter Number  :");
        iValue=sobj.nextInt();

       Logic lobj=new Logic() ;

       lobj.CheckDivisible(iValue);
    }
}